testplan/exporters/testing/pdf/__init__.py

```diff
@@ -213,18 +220,32 @@ class PDFExporter(Exporter):
 
     CONFIG = PDFExporterConfig
 
-    def export(self, source):
-
-        pdf_path = self.cfg.pdf_path
+    def __init__(self, **options):
+        super(PDFExporter, self).__init__(**options)
+        self.name = "PDF exporter"
```

Comment:
I think we could expose name config option to user, and give it a default if user didn't specify.